Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Collections/Editor/BaseCollectionEditor.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEditor;
+
+
+namespace Microsoft.MixedReality.Toolkit.SDK.UX.Collections
+{
+    [CustomEditor( typeof(BaseCollection), true )]
+    public class CollectionEditor : Editor
+    {
+        public override void OnInspectorGUI()
+        {
+            // Draw the default
+            base.OnInspectorGUI();
```

Comment:
Let's go ahead and grab all the properties off the object so we can have a clean inspector.

Reply:
Not sure what you mean here. I'm just adding a custom button.

Reply:
Just grab all the serialized properties, and implement them in the gui update. 

If you'd like I can open a PR that does this, if it's still unclear.